.github/workflows/dev-deploy-closed-prs.yml

```
@@ -0,0 +1,88 @@
+      - name: Set image tag based on date and run number
+        id: set-tag
+        run: echo "image-tag=$(date +'%Y-%m-%d')-${{ github.run_number }}" >> $GITHUB_OUTPUT
+
+      - name: Run tests
+        working-directory: ./server
+        run: |
+          chmod +x ./gradlew
+          ./gradlew test
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)